admin/worker/run_autoscaler.go

```diff
@@ -72,7 +72,20 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 			continue
 		}
 
-		w.logger.Info("succeeded in autoscaling:", zap.String("project_id", updatedProject.Name), zap.Int("project_slots", updatedProject.ProdSlots))
+		logInfo := []zap.Field{
+			zap.String("project_name", updatedProject.Name),
+			zap.Int("updated_slots", updatedProject.ProdSlots),
+			zap.Int("prev_slots", targetProject.ProdSlots),
+		}
+
+		updatedProjectOrg, err := w.admin.DB.FindOrganization(ctx, updatedProject.OrganizationID)
+		if err != nil {
+			w.logger.Error("failed to find org for the project", zap.String("project_name", updatedProject.Name), zap.String("org_id", updatedProject.OrganizationID), zap.Error(err))
+		} else {
+			logInfo = append(logInfo, zap.String("org_name", updatedProjectOrg.Name))
+		}
```

Comment:
Move finding of the org up to after the call to `FindProject`, before calling `UpdateProject`. That way you can just call `continue` if there's an `err`, and the code will be cleaner.

Reply:
Sounds good!